stdlib/CompilerSupportLibraries_jll/src/CompilerSupportLibraries_jll.jl

```diff
@@ -60,6 +60,15 @@ function __init__()
     global libgomp_path = dlpath(libgomp_handle)
 end
 
+# JLLWrappers API compatibility shims.  Note that not all of these will really make sense.
+# For instance, `get_artifact_dir()` won't actually be the artifact directory, because
+# there isn't one.  It instead returns the overall Julia prefix.
 is_available() = true
+get_artifact_dir() = artifact_dir
```

Comment:
Thanks @staticfloat I appreciate it! I'll have a closer look later and will also test it, but one quick remark:

```suggestion
find_artifact_dir() = artifact_dir
```
And similarly elsewhere.